src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
     let device = this.devices[address];
 
     if (device === undefined) {
-      device = openSync(getBoardRevision() === VERSION_1_MODEL_B_REV_1 ? 0 : 1);
+      if (busNumber === undefined) {
+        busNumber = this.busNumber;
+      }
+      device = openSync(busNumber);
       this.devices[address] = device;
     }
 
     return device;
   }
 
+  public configure(options?: any): void {
```

Comment:
The issue is that if you want to use the 2 i2c to read different infos, the only place where you can do it is in ```i2cConfig``` Where you can "map" an address with a bus. This is what you do in ```getDevice```. This is the trick that i use to make it work.
Seeing the structure off johnny five (which i use on top of raspi) this is the only way to do it. Johnny Five does not use an I2C instance like it does with five.Led. Which means that you can't really have multiple ones. Instead you use ```board.i2cRead``` only knowing the address you are reading at. Consequently i used the fact that you map one device for one address in ```getDevice```.
The trick is then to know the busNumber when you ```openSync```. That's why i added busNumber as an optional argument to ```getDevice```. When not supplied it will behave exactly as it did without that argument. 
Then in ```i2cConfig``` i supply the ```bus``` but i had to then call a ```getDevice``` with that ```bus``` to make sure we would use the right ```busNumber``` with ```openSync```. That's why i created ```configure```